fabric/README.md

```diff
@@ -47,26 +47,21 @@ The requirements are:
 * your Service Provider ID (SPID)
 
 In order to use Intel's Attestation Service (IAS) you need to register
-with Intel. [Here](https://software.intel.com/en-us/articles/code-sample-intel-software-guard-extensions-remote-attestation-end-to-end-example)
-you can find more details on how to obtain a signed client certificate,
-registering it and get a SPID.
-
-We currently make use of `unlinkable signatures` for the attestation, thus, when registering with the IAS please choose
-unlinkable signatures. In the case you prefer linkable attestation,
-e.g., because you already have linkable IAS EPID credentials, change
-in line 217 of [../ecc_enclave/sgxcclib/sgxcclib.c](../ecc_enclave/sgxcclib/sgxcclib.c)
-the constant `SGX_UNLINKABLE_SIGNATURE` to `SGX_LINKABLE_SIGNATURE`,
-re-compile and re-deplay [ecc_enclave](../ecc_enclave#build) and [ecc](../ecc#getting-started),
-configure your IAS settings as above with your linkable credentials and run the auction example as follows.
+with Intel. [Here](https://software.intel.com/en-us/form/sgx-onboarding)
+you can find more details on how to register and optain your SPID plus corresponding api-key.
+
+We currently support both `linkable' and 'unlinkable' signatures for the attestation.
+The type of attestation used is selected based on your 'ECC_ATTESTATION_TYPE' environment variable:
+'epid_unlinkable' for unlinkable or 'epid_linkable' for linkable signatures. The default in case you 
+do not define that environment variable is 'epid_unlinkable'.
```

Comment:
Maybe rewording? By default `ECC_ATTESTATION_TYPE` is set to `epid_unlinkable`

Reply:
we do not set the environment variable, just set a default build variable if env variable is absent. did rephrase though a bit